src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1772,16 +1772,23 @@ private List<IEnumerable<Table.TableValueHolder>> GetRevisionsTables(DocumentsOp
             {
                 foreach (var collection in collections)
                 {
-                    IEnumerable<Table.TableValueHolder> table = null;
-                    var collectionName = _documentsStorage.GetCollection(collection, throwIfDoesNotExist: true);
+                    CollectionName collectionName = null;
+                    try
+                    {
+                        collectionName = _documentsStorage.GetCollection(collection, throwIfDoesNotExist: true);
```

Comment:
let's pass `throwIfDoesNotExist: false` and check for `null`
also, we check the collection of the _documents_, but then try to open a revision table. 
So, can it be that we have no table for documents, but do have a one for revisions?

Reply:
You are right.
I changed it, now we only check in the RevisionnStorage..